source/includes/administration/_pricings.md

```diff
@@ -0,0 +1,233 @@
+
+
+<!-------------------- GET PRICING -------------------->
+#### Retrieve a pricing
+
+`GET /pricings/:id`
+
+Retrieve a pricing's details.
+
+```shell
+# Retrieve an organization
+curl "https://cloudmc_endpoint/rest/pricings/03bc22bd-adc4-46b8-988d-afddc24c0cb5" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "supportedCurrencies": [
+      "CAD"
+    ],
+    "productCatalogs": [
+      {
+        "mode": "ALL_CONNECTIONS_OF_TYPE",
+        "serviceType": "cloudca",
+        "connectionIds": [],
+        "name": {
+          "en": "Default cloud.ca",
+          "fr": "Défaut cloud.ca"
+        },
+        "changes": [],
+         "description": {
+          "en": "cloud.ca default product catalog",
+          "fr": "catalogue des produits cloud.ca par défaut"
+        },
+        "id": "5ba5df7c-1e60-4d99-869d-dd3d97826b2e",
+        "categories": [
+          {
+            "name": {
+              "en": "Compute",
+              "fr": "Compute"
+            },
+            "id": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
+          }
+        ],
+        "products": [
+          {
+            "metricType": "GAUGE",
+            "unit": {
+              "unit": "HOUR",
+              "name": {}
+            },
+            "period": "HOUR",
+            "name": {
+              "en": "vCPU",
+              "fr": "vCPU"
+            },
+            "transformer": {
+              "type": "PROPORTIONAL_TO_TIME"
+            },
+            "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
+            "attribute": "RAWUSAGE",
+            "source": "1",
+            "filters": [],
+            "sku": "vCPU",
+            "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
+          }
+        ]
+      }
+    ],
+    "organization": {
+      "id": "409c1162-a930-4207-bdf9-710580c3542b"
+    },
+    "pricingProducts": [
+      {
+        "unitPrice": 123.0,
+        "product": {
+          "metricType": "GAUGE",
+          "unit": {
+            "unit": "HOUR",
+            "name": {}
+          },
+          "period": "HOUR",
+          "name": {
+            "en": "sfg",
+            "fr": "sdfg"
+          },
+          "transformer": {
+            "type": "PROPORTIONAL_TO_TIME"
+          },
+          "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
+          "attribute": "RAWUSAGE",
+          "source": "1",
+          "filters": [],
+          "sku": "sdfg",
+          "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
+        },
+        "cogs": 321.0,
+        "currency": "USD",
+        "id": "45425fe6-beb3-4c78-be68-f82eeb3976c6"
+      }
+    ],
+    "name": {
+      "en": "testen",
+      "fr": "testfr"
+    },
+    "description": {
+      "en": "descen",
+      "fr": "testfr"
+    },
+    "id": "096bbfee-a6e6-4373-8a4e-1a8b0f2bf484",
+    "effectiveDate": "2020-05-08T16:53:45Z"
+  }
+}
+```
+
+Attributes | &nbsp;
```

Comment:
the same comments as above

Reply:
f428ed5b76ae49b8bf4e644a9c5cd49c51399110